.travis.yml

```diff
@@ -7,9 +7,10 @@ env:
     - REPO=docksal/varnish
 
   matrix:
-    - VERSION=6.1 TAGS=6.1,6,latest
-    - VERSION=5.2 TAGS=5.2,5
-    - VERSION=4.1 TAGS=4.1,4
+    - VERSION=6.1.1 TAGS=6.1,6,latest
+    - VERSION=6.0.2 TAGS=6.0
+    - VERSION=5.2.1 TAGS=5
```

Comment:
Drop 5.2.1 since it's no longer supported (retired), according to
https://varnish-cache.org/releases/